packages/dashboards/examples/src/components/app/prototype-index.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component } from "@angular/core";
```

Comment:
Component for easy access to prototypes and test pages similar to what we have for the charts demo app.